js/zb.js

```diff
@@ -2446,6 +2447,77 @@ module.exports = class zb extends Exchange {
         return result;
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        let side = undefined;
+        if (fromAccount === 'spot' || toAccount === 'futures') {
```

Comment:
Let's use `toAccount === 'future'` (not future**s**) to align it with the market flags and market types.

Reply:
@samgermain yes, if we could support both the unified type and the exchange-specific type – that would be ideal.